ultravox/inference/infer_test.py

```diff
@@ -14,13 +14,15 @@
 @pytest.fixture(scope="module")
 def tokenizer():
     return transformers.AutoTokenizer.from_pretrained(
-        "meta-llama/Meta-Llama-3-8B-Instruct"
+        "../tests/assets/hf/Meta-Llama-3-8B-Instruct"
```

Comment:
probably don't need to have this path at the root. Suggest using
`assets/hf/foo` (i.e., put the assets folder under the top-level `ultravox`)